packages/origin-ui-core/src/components/Table/PaginatedLoaderHooks.tsx

```diff
@@ -205,10 +206,13 @@ export function checkRecordPassesFilters(
 
     for (const filter of filters) {
         const filteredPropertyResolvedValue = filter.property(record);
-
         if (typeof filteredPropertyResolvedValue !== 'undefined') {
             switch (filter.input.type) {
                 case CustomFilterInputType.string:
+                    console.log(
+                        '>>> String filter: record to filter:',
+                        filteredPropertyResolvedValue
+                    );
```

Comment:
Do we need to keep these console logs?

Reply:
sorry. I am just keep to forget to remove consols. May be we should add eslint rule which forbids them

Reply:
We thought about it, but we do have intentional console.logs in the codebase, so it wouldn't work.